src/ext/java/org/opentripplanner/ext/transmodelapi/model/framework/TriangleFactorsInputType.java

```diff
@@ -0,0 +1,31 @@
+package org.opentripplanner.ext.transmodelapi.model.framework;
```

Comment:
This should be moved to the `model.plan` package. The packages follows the Netex standard - so I understand that it is not obvious where to put things.

Reply:
Done.